jinja2_gcp_secret_manager/__init__.py

```diff
@@ -36,6 +34,8 @@ def parse(self, parser):
             parser.stream.skip(1)
             project = parser.parse_expression()
 
+        if not project.value:
+            parser.fail("Project not specified", lineno=lineno)
```

Comment:
This project check needs to go after parsing the tag so that project can be set via the tag as well.  For Application Default Credentials, `PROJECT_ID` will not be set as ADC is not inherently tied to any one project (as a service account is).  With the check here, we can still use the default `PROJECT_ID` if it's found, but we allow for overriding it on the tag level with a custom project ID.

Reply:
Works

Reply:
Modified that slightly in my last commit as well.  Found a bug while testing it where if you pass an unset variable to `project=` it would error out in a weird way.  So, the update will check the type is correct before checking `project.value` now.  Wrong type (i.e. bad syntax) or no value will now trigger the error.

Reply:
Okay